WTWSMS/events/religious_events.txt

```diff
@@ -3634,11 +3680,11 @@ character_event = {
 				trait = zealous
 			}
 			modifier = {
-				factor = 3
+				factor = 5
```

Comment:
Cynics could care less and are more willing to convert as a matter of convenience if it can yield an increase of their power. Compared with other issues and modifiers, this is more balanced and realistic.